__tests__/reorder-slide.test.js

```diff
@@ -0,0 +1,184 @@
+const PPTX = require('../index.js');
+const fs = require('fs');
+const tmpDir = `${__dirname}/tmp`;
+
+describe('Presentation Module', () => {
+    test("should be able to load an existing PPTX (containing three slides) and swap slide #'s 2 and 3", async () => {
+        try {
+            let testFileName = 'presentation-swap-existing-slides-small.pptx';
+
+            expect.assertions(1);
+
+            if (fs.existsSync(`${tmpDir}/${testFileName}`)) {
+                fs.unlinkSync(`${tmpDir}/${testFileName}`);
+            }
+
+            let pptx = new PPTX.Composer();
+
+            await pptx.load(`${__dirname}/fixtures/three-slides.pptx`);
+            await pptx.compose(async pres => {
+                let slide = pres.getSlide(3);
+                slide.moveTo(2);
+
+                slide.addText(text => {
+                    text
+                        .value('If this slide says "original slide #3" but is in slide position 2, then it worked.')
```

Comment:
Are you manually validating that the slide text is correct? Is there no way to read the values of the slide?

Reply:
Technically, yes, we could load up an already correct version of the file into an XML string, and just compare the XML contents, here's the problem: as we add additional features to node-pptx, the XML changes, and then that invalidates the test. I've been holding off on doing direct comparisons like this because I don't want to end up changing a bunch of tests after every new feature.

Reply:
Ok understood, but if you can "select" a slide to modify can't you check it's content?

Reply:
You _could_ load a correct file, pick out that slide, and do a text comparison, yeah. I could add that if you want...

Reply:
I think that is the correct way to test this functionality thanks!

Reply:
Done. See new stuff in reorder-slide.test.js.